Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -3,6 +3,12 @@ import Library
 import Prelude
 import UIKit
 
+private enum Layout {
+  enum Sheet {
+    static let offset: CGFloat = 200
```

Comment:
This fine for different device sizes?

Reply:
Yeah, it looks like we need enough space to still show the shipping location cell behind the modal.